### ISSUE TITLE
Problem with prediction from checkpoint

### ISSUE THREAD

**Issue by adiyoss** [thtrieu/darkflow#142]
Hey,
I'm trying to classify new images after fine-tune yolo on my dataset, but somehow I always get an empty prediction (i.e. the model didn't find anything), on both test and train sets.
During training I see that the loss is decreasing nicely.

Do you have any idea what can cause this wrong predictions? 
I use the following command:

./flow --model exp/exp-1-sur/cfg/yolo-v2-car.cfg --load -1 --test ~/dataset/video-inform/04-04-17-cars/sur/test/ --backup exp/exp-1-sur/ckpt/ --gpu 1.0 --batch 1 --json

Thanks

**Comment by ggookey123** [comment]
I have same problem. Anyone have solution?

**Comment by jcarletgo** [comment]
Related: Issue #80

**Comment by dkarmon** [comment]
same problem here - solutions suggested at [#80](https://github.com/thtrieu/darkflow/issues/80) did not solve it. 
Please advise

**Comment by nuitvolgit** [comment]
I added following single code on line 78 of darkflow/net/flow.py
...
Line 77: feed_dict = {self.inp : this_inp}
Line 78: feed_dict.update(self.feed) 				# <= Added this line
...

I don't exactly know why but it works for me.
FYI, the weights were trained from scratch with no pre-training.

**Comment by guyzigdon** [comment]
@nuitvolgit did you train again after adding this line? or just tested?

**Comment by nuitvolgit** [comment]
I added the line and trained from scratch.
